megaavr/extras/LowPower.md

```diff
@@ -0,0 +1,635 @@
+# LowPower.cpp
+    This document is a comprehensive list of all the functions from LowPower.cpp. Each function includes descriptions for how they change the hardware to decrease the power draw for DxCore microcontrollers.
+    All information for the hardware changes and interrupts can be found in the AVR128DA48 datasheet from Microchip which can be found here: https://www.microchip.com/en-us/product/AVR128DA48
```

Comment:
Not quite sure why but GitHub is rendering all of the indented sections as code blocks, let's fix that so these sections display as paragraphs. 
```suggestion
This document is a comprehensive list of all the functions from LowPower.cpp. Each function includes descriptions for how they change the hardware to decrease the power draw for DxCore microcontrollers.
All information for the hardware changes and interrupts can be found in the AVR128DA48 datasheet from Microchip which can be found here: https://www.microchip.com/en-us/product/AVR128DA48
```